docker/daemon.go

```diff
@@ -141,7 +153,17 @@ func mainDaemon() {
 	job.Setenv("TlsCert", *flCert)
 	job.Setenv("TlsKey", *flKey)
 	job.SetenvBool("BufferRequests", true)
-	if err := job.Run(); err != nil {
-		log.Fatal(err)
+	err = job.Run()
```

Comment:
Just didn't clean up from a slightly different implementation, I'll take care of it.